scripts/core/activity/activity.js

```diff
@@ -172,6 +173,18 @@ function SuperdeskProvider($routeProvider, _) {
                 return privileges.userHasPrivileges(activity.privileges);
             }
 
+            function checkActivityEnabled(activity) {
+                if (!_.get(config, 'activity')) {
+                    return true;
+                }
+
+                if (_.isUndefined(config.activity[activity._id])) {
```

Comment:
maybe it's better the way you have it, it would be too much config once you put there anything